docs/guides/divergence.md

```diff
@@ -0,0 +1,89 @@
+# Handling divergent changes
+
+## What are divergent changes?
+
+A [divergent change][glossary_divergent_change] occurs when multiple visible
+commits have the same change ID.
+
+Normally, when commits are rewritten, the original version (the "predecessor")
+becomes hidden and the new commit (the "successor") is visible. Thus, only one
+commit with a given change ID is visible at a time.
+
+But, a hidden commit can become visible again. This can happen if:
+
+- A visible descendant is added. For example, `jj new REV` will make `REV`
+  visible even if it was hidden before.
+
+- It is made the working copy. `jj edit REV` will make `REV` visible if it
+  wasn't already.
+
+Divergent changes also occur if two different users or processes amend the same
+change, creating two visible successors. This can happen when:
+
+- Multiple people edit the same change simultaneously in different repositories.
```

Comment:
nit: expand this to mention to mention a coworker working on the same branch and then re-pushing, since many don't understand that as a rewrite

Reply:
Are you referring to a coworker appending a new commit to your branch? Do you think [this suggestion would address that](https://github.com/jj-vcs/jj/pull/7061#discussion_r2229505692) too?

Reply:
I mean this case here: https://github.com/jj-vcs/jj/discussions/7004

Reply:
that should be good enough